src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -169,6 +169,21 @@ function build_adios2
         cd $ADIOS2_SRC_DIR || error "Can't cd to $ADIOS2_SRC_DIR"
         info "Configuring ADIOS2-$bt (~1 minute)"
 
+        if [[ "$bt" == "par" ]]; then
+            # Change all references from adios2 to adios2_mpi.
+            find . -name "CMakeLists.txt" -exec sed -i "s/adios2/adios2_mpi/g" {} \;
```

Comment:
Was this easier than creating a patch that changes the generated library name for the parallel version?

Reply:
IMHO, the `sed` approach is potentially more version agnostic. As changes are made to those files, it is more likely to still work. OTOH, when/if it fails, it will likely be harder to diagnose than an error message from build_visit indicating a required patch had failed.